apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -779,6 +780,28 @@ export function createApi(
 
       return statements;
     },
+    loadStatementsAndUsers: async (
+      networkId: NetworkID,
+      spaceId: string,
+      userIds: string[]
+    ): Promise<{ statements: Statement[]; users: User[] }> => {
+      const { data } = await apollo.query({
+        query: STATEMENTS_AND_USERS_QUERY,
+        variables: {
+          where: {
+            delegate_in: userIds,
+            network: networkId,
+            space: spaceId
+          },
+          userIds
+        }
+      });
```

Comment:
```suggestion
      const { data } = await apollo.query({
        query: STATEMENTS_AND_USERS_QUERY,
        variables: {
          statementsWhere: {
            delegate_in: userIds,
            network: networkId,
            space: spaceId
          },
          userIds
        }
      });
```

Right now we are not filtering statements at all because argument name doesn't match, so we just fetch random 20 statements that are most likely not for the members of the space.